src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -100,7 +113,16 @@ public void start() {
       addBombardmentSources();
       m_needToAddBombardmentSources = false;
     }
+    fightCurrentBattle();
+    if (rocketHelper != null && !games.strategy.triplea.Properties.getRollRocketsAfterSBR(getData())) {
+      rocketHelper.fireRockets(m_bridge, m_bridge.getPlayerID());
+      rocketHelper = null;
```

Comment:
Because the official rules for Axis and Allies have this stupid idea that the rocket damage should be applied before the strategic bombing target units are decided (but not the target territories).

It not being null means that it has not been processed yet. I thought it was pretty obvious.